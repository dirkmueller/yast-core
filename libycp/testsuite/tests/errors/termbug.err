Parsed:
----------------------------------------------------------------------
{
    // term contents
    // filename: "tests/errors/termbug.ycp"
    term contents = `VBox ();
    contents = add (contents, `VSpacing (1));
    return contents;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // term a
    // filename: "tests/errors/termbug.ycp"
    term a = { return y2error ("hell"); };
    return a;
}
----------------------------------------------------------------------
