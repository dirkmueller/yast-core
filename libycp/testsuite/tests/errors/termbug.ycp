// termbug
{
    term contents = `VBox ();
    contents = add (contents, (`VSpacing (1)));

    return contents;
}

{
    term a=``(y2error("hell"));

    return a;
}
