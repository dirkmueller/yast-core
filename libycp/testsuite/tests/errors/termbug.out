(`VBox (`VSpacing (1)))
(y2error ("hell"))
